config/software/dep-selector-libgecode.rb

```diff
@@ -19,11 +19,13 @@
 
 license "Apache-2.0"
 license_file "https://github.com/chef/dep-selector-libgecode/blob/master/LICENSE.txt"
+source git: "https://github.com/chef/dep-selector-libgecode.git"
```

Comment:
This shouldn't be needed once the dep-selector-libgecode gem is released right?  Doing just gem install like we did before should continue working as soon as https://github.com/chef/dep-selector-libgecode/pull/49 is actually pushed to rubygems.